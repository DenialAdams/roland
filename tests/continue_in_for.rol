proc main() {
   let x: u8 = 3;
   for i in 0..x {
      if i == 1 {
         // i still needs to increment to avoid an infinite loop
         continue;
      }
      println("Hello from the loop");
   }
}
