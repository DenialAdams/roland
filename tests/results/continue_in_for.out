Hello from the loop
Hello from the loop
